tmp/ambient_occlusion.cpp

```diff
@@ -1,144 +0,0 @@
-// TODO: Fix the npe_arg for igl::AABB and std::function
```

Comment:
Are we keeping any version of this function or just killing all of it? The VFPN version (first function) is probably useful and easy to bind.

Reply:
Resolved this in the commit this morning. Now there is only the VFPN function, and  I have removed the file from tmp directory.